packages/botonic-react/src/components/index-types.ts

```diff
@@ -151,9 +152,12 @@ export interface ThemeProps extends StyleProp {
     enableArrows?: boolean
   }
   reply?: StyleProp & CustomProp
-  triggerButton?: { notificationsEnabled?: boolean } & ImageProp &
-    StyleProp &
-    CustomProp
+  triggerButton?: ImageProp & StyleProp & CustomProp
+  notifications?: EnableProp & {
+    banner?: CustomProp & EnableProp & { text?: string }
+    triggerButton?: EnableProp
+  }
+  scrollButton?: EnableProp & CustomProp
```

Comment:
Doubt: Why not adding the `scrollButton` inside the `notifications` prop?

Reply:
Even if the notifications are not activated, the scroll behaviour will be affected and I think it might be interesting to activate this button when the notifications are deactivated.

Reply:
We can talk on Monday and decide how best to activate this button.

Reply:
Applied but I would like to talk about how to activate all this on Monday.